admin/packaging.py

```diff
@@ -606,6 +606,11 @@ def __str__(self):
         # E.g.
         # /opt/flocker/lib/python2.7/site-packages/sphinx/locale/bn/LC_MESSAGES/sphinx.mo
         'file-not-in-%lang',
+
+        # Twisted 16.6 includes an executable C source file.
```

Comment:
It's part of Twisted's MS Windows event loop.
Nothing to do with Flocker, but it gets installed when we install the Twisted package.
And unfortunately triggers our rpmlint packaging checks.